src/pages/SignupPage.jsx

```diff
@@ -3,30 +3,30 @@ import React from 'react';
 import { Layout } from '../components';
 import { Input } from '../components/Inputs';
 import { CustomButton } from '../components/Button/CustomButton';
-import { SignUpTheme, SingUpTeme } from '../styles/themes/CustomSingUpPage';
+import { LoginPageTheme } from '../styles/themes/CostomLogInPage';
 
 import { Grid } from '@mui/material';
 import Typography from '@mui/material/Typography';
 import Link from '@mui/material/Link';
-import {useStyles} from './SingUpPageCustomInput';
-import { width } from '@mui/system';
+// import {useStyles} from './SingUpPageCustomInput';
+// import { width } from '@mui/system';
 
 export const SignupPage = () => {
-  const classes = useStyles();
+  // const classes = useStyles();
   return (
     <Layout>
       <Grid container direction="column" alignItems="center" style={{ marginTop: '10vmin' }} gap="2rem">
-        <Typography theme={SignUpTheme} variant="h2" component="h2" color="#16BAC6" fontSize="2.8rem">
+        <Typography theme={LoginPageTheme} variant="h2" component="h2" color="#16BAC6" fontSize="2.8rem">
           Create your account
         </Typography>
         <Grid container justifyContent="center" gap="3rem">
-          <Input label="first name" type='text' />
-          <Input label="last name" type='text' />
+          <Input label="first name" type="text" />
+          <Input label="last name" type="text" />
         </Grid>
-        <Input label="email" type='email' />
-        <Input label="phone number" type='tel' />
-        <Input label="password" type='password' />
-        <Typography theme={SignUpTheme}>
+        <Input label="email" type="email" />
+        <Input label="phone number" type="tel" />
+        <Input label="password" type="password" />
```

Comment:
If Marta will merge her branch, then you can add "fullWidth" parameter here. The inputs will then be on the entire length
e.g 
```js
        <Input label="email" type="email" fullWidth />
```